hr_skill/views/hr_employee_skill.xml

```diff
@@ -0,0 +1,74 @@
+<odoo>
+  
+    <record id="view_hr_employee_skill_form" model="ir.ui.view">
+        <field name="name">hr.employee.skill.form</field>
+        <field name="model">hr.employee.skill</field>
+        <field name="arch" type="xml">
+            <form string="Employee Skill">
+                <group>
+                    <group>
+                        <field name="employee_id"/>
```

Comment:
I don't think this field is needed. I think it is confusing for the user. 
If  add a skill on an employee form I would think it would be for that employee and I now randomly can choose another employee. 

Or is there another reason for this field?

Reply:
This field is shown for another reason on that form, form itself should not be shown. Fixed in other way 👍